pandas/tests/frame/methods/test_replace.py

```diff
@@ -1363,3 +1363,27 @@ def test_replace_after_convert_dtypes(self):
         result = df.replace(1, 10)
         expected = pd.DataFrame({"grp": [10, 2, 3, 4, 5]}, dtype="Int64")
         tm.assert_frame_equal(result, expected)
+
+    @pytest.mark.parametrize(
+        "df, to_replace",
+        [
+            (
+                {"one": ["a", "b ", "c"], "two": ["d ", "e ", "f "]},
```

Comment:
Since `df` is the same all of the time no need to parametrize; just place in the body